src/Agent.ts

```diff
@@ -65,6 +65,9 @@ export class Agent {
     if (error["__trackjs__"]) {
       return false;
     }
+    if (!(error as any instanceof Error)) {
+      error = new Error("" + error);
+    }
```

Comment:
We might want to considering porting our full-featured serialize function over from the browser agent, but that felt like overkill for now. There are less bad-practices around errors in the Node ecosystem 🤞